apps/meteor/app/lib/server/methods/setUsername.ts

```diff
@@ -10,13 +10,6 @@ import { checkUsernameAvailability } from '../functions';
 import { RateLimiter } from '../lib';
 import { saveUserIdentity } from '../functions/saveUserIdentity';
 
-declare module '@rocket.chat/ui-contexts' {
```

Comment:
this was recent added so I guess we should add it back

Reply:
hm, would be cool to know why. I moved the declaration to the actual type on UI-context package.